app/assets/javascripts/import-progress.js

```diff
@@ -0,0 +1,23 @@
+$(function updateImportProgress() {
```

Comment:
We can use the framework in govuk_admin_template for this, for consistency with other publishing apps:

https://github.com/alphagov/govuk_admin_template/blob/master/JAVASCRIPT.md